src/vs/workbench/contrib/tasks/common/taskConfiguration.ts

```diff
@@ -1635,7 +1635,7 @@ namespace CustomTask {
 			{
 				name: configuredProps.configurationProperties.name || contributedTask.configurationProperties.name,
 				identifier: configuredProps.configurationProperties.identifier || contributedTask.configurationProperties.identifier,
-				icon: contributedTask.configurationProperties.icon
+				icon: configuredProps.configurationProperties.icon
```

Comment:
Should this be this to match the other lines?

```ts
configuredProps.configurationProperties.icon || contributedTask.configurationProperties.icon
```

Reply:
Don't think the contributed task will ever have an icon until we have API